kalite/distributed/features/superuser_create.feature

```diff
@@ -5,55 +5,51 @@ Feature: Create superuser from the in browser modal
         Then there should be no modal displayed
 
     Scenario: Superuser does not exist yet
-        Given superuser is deleted
-        And I am on the homepage
-        Then I should see a modal
+        Given I am on the homepage
+        And superuser is deleted
```

Comment:
I would reverse the order here: 
```gherkin
Given superuser is deleted
And I am on the homepage
```

Transaction errors seem to occur if you attempt to access the database (as with an API endpoint) while a write is still happening; I think that's the case here. Reversing the order should ensure the write finishes before any API endpoints are called.